dot/parachain/types/types.go

```diff
@@ -757,7 +757,32 @@ type BackedCandidate struct {
 	// The validity votes themselves, expressed as signatures.
 	ValidityVotes []ValidityAttestation `scale:"2"`
 	// The indices of the validators within the group, expressed as a bitfield.
-	ValidatorIndices BitVec `scale:"3"` // TODO: it's a bitvec in rust, figure out actual type
+	ValidatorIndices BitVec `scale:"3"`
+}
+
+func NewBackedCandidate(
+	candidate CommittedCandidateReceipt,
+	validityVotes []ValidityAttestation,
+	validatorIndices []bool,
+	coreIndex *CoreIndex,
+) (*BackedCandidate, error) {
+	const maxCoreIndex uint32 = 255 // math.MaxUint8
```

Comment:
so why not using math.MaxUint8?

Reply:
As we don't need a math Package anywhere in the file, I have created a constant for maxUint8 value and used it directly that avoids small overhead of importing math.